packages/amplify-category-auth/provider-utils/awscloudformation/triggers/PreSignup/cloudformation-templates/PreSignup.json.ejs

```diff
@@ -102,11 +102,11 @@
                     "REGION": { 
                         "Ref": "AWS::Region"
                     },
-                    "DOMAINWHITELIST": {
-                        "Ref": "DOMAINWHITELIST"
+                    "DOMAINALLOWLIST": {
```

Comment:
I think we'll need to be careful with these env variables. Changing them could break existing lambda triggers